src/libs/auth.js

```diff
@@ -324,53 +321,25 @@ authStore.subscribe(withErrorReporting('Error checking registration', async (sta
 
 authStore.subscribe(withErrorReporting('Error checking TOS', async (state, oldState) => {
   if (!oldState.isSignedIn && state.isSignedIn) {
-    const tosDetails = await Ajax().User.getTermsOfServiceDetails()
-    const termsOfService = parseToSDetails(tosDetails)
+    const tosDetails = await Ajax().User.getTermsOfServiceAdherenceStatus()
```

Comment:
I'd rename this (and the rest of the instances) `getTermsOfServiceComplianceStatus` to match the naming changes in your Sam PR!